packages/@uppy/companion/src/standalone/helper.js

```diff
@@ -113,6 +115,13 @@ const getConfigFromEnv = () => {
     maxFileSize: process.env.COMPANION_MAX_FILE_SIZE ? parseInt(process.env.COMPANION_MAX_FILE_SIZE, 10) : undefined,
     chunkSize: process.env.COMPANION_CHUNK_SIZE ? parseInt(process.env.COMPANION_CHUNK_SIZE, 10) : undefined,
   }
+
+  // todo remove COMPANION_S3_GETKEY_SAFE_BEHAVIOR in next major and use this getKey implementation instead by default
+  if (process.env.COMPANION_S3_GETKEY_SAFE_BEHAVIOR === 'true') {
+    envConfig.providerOptions.s3.getKey = (req, filename) => `${uuid.v4()}-${filename}`
```

Comment:
Might as well align with Tus and just do a random name?

```suggestion
    envConfig.providerOptions.s3.getKey = (req, filename) => uuid.v4()
```

Reply:
What's the benefit of that? I thought it would be nice to include the file name to make it easier to find the uploaded file